src/concepts/external_data_integration.md

```diff
@@ -0,0 +1,75 @@
+
+## Setting up connectors
+
+This is a step-by-step guide for setting up a new connector in Fivetran, scheduling it via Airflow and transforming imported data via bigquery-etl:
+
+1. Log in to [Fivetran](https://fivetran.com) and select the environment the new connector should be created in.
+1. Click on the "Add Connector" button and search for the service data that should be imported from. If no connector is available for the service please check out ["Developing custom connectors"](#developing-custom-connectors) for writing custom connectors.
+1. Specify the name of the destination dataset Fivetran will dump data into.
+1. Configure the connector by providing API credentials and other required settings.
+1. Finish the setup and wait for the connection tests to pass.
+1. Once the connector has been created a historical data import will be automatically triggered. This import can take a few minutes up to several days depending on how much historical data is available.
+1. Specify the data transformation in [bigquery-etl](https://github.com/mozilla/bigquery-etl) or [private-bigquery-etl](https://github.com/mozilla/private-bigquery-etl) when working with sensitive data. Please follow the [general guide of creating a derived dataset](https://mozilla.github.io/bigquery-etl/cookbooks/creating_a_derived_dataset/). When writing the query, data will be queried from the dataset that was specified when the connector was set up. The connector documentation specifies what tables will be available and what the schemas will look like.
+   - Make sure to query from the correct data destination depending on the environment for which the connector was set up for.
+1. [Scheduling](https://github.com/mozilla/bigquery-etl/blob/main/docs/reference/scheduling.md) the ETL and the Fivetran import is also done in bigquery-etl. In the `metadata.yaml` file of the transformation queries. In addition to specifying the DAG, also specify the Fivetran tasks that the ETL depends on, for example:
+
+   ```yaml
+   scheduling:
+     dag_name: bqetl_external_service
+     depends_on_fivetran:
+       - task_id: fivetran_import_1
+       - task_id: another_fivetran_import
+   ```
+
+1. Once the changes have been merged into bigquery-etl and the Airflow DAGs have been updated and are available in Airflow the connector ID needs to be configured in the Airflow Admin interface. Looking at the Airflow DAGs each Fivetran import task will reference an Airflow variable. To configure these variables in the [Airflow Admin - Variables settings](https://workflow.telemetry.mozilla.org/variable/list/) add a new entry. The **Key** needs to be set to the variable name as shown in the DAG source, the **Value** is the Fivetran connector ID which can be copied from the Connection Details of the Fivetran connector in the "Setup" tab.
+1. Enable the DAG in Airflow and trigger an import.
```

Comment:
Might be worth mentioning that the first Airflow run will turn off the connector's default Fivetran scheduling.

Reply:
I agree with this, it can be a bit confusing. I remember when I was working with a connector I was trying to find the option in the Fivetran UI and wasn't immediately apparent to me that this is some sort of "magic" that Fivetran does once you trigger a run manually.